releasetasks/templates/source.yml.tmpl

```diff
@@ -72,3 +73,32 @@
             build:
                 platform: none
         #}
+-
+    taskId: "{{ stableSlugId(buildername_signing) }}"
```

Comment:
Yeah, that's quite similar to how we iterate over en-US platforms before including funsize, I think? I'm OK either way here, maybe it's better to wait until after we have other similar signing tasks to see if it makes sense to refactor.

I don't think the problem with "all deliverables ready" changes even if you factor out signing task to its own template.